app/views/tag/_subscribe_button.html.erb

```diff
@@ -24,10 +24,26 @@
 </style>
 
 <script type="text/javascript">
+  
   $('#popover_button').popover({
   	content: $('#mypopcontent').clone().removeClass('hidden'),
   	placement: 'bottom',
   	html: true,
 
+  }).parent().on('click', '#mypopcontent', function(event) {
+        event.stopPropagation();
+  });
+
+  $('#popover_button').popover({
+  	content: $('#mypopcontent').clone().removeClass('hidden'),
+  	placement: 'bottom',
+  	html: true,
+
+  }).click(function(e) {
+    e.stopPropagation();
+  });
+
+  $(document).click(function(e){
```

Comment:
I think this is the only relevant code. Why do we need lines 33-45? Can you please check this and remove what we don't need?

Reply:
line 46,47 hides the popover whenever we click anywhere on the document. However, this is not the desired functionality. We want to exclude the popover window itself and also the subscribe button from this. (we don't want to close the popover when we 1. click on subscribe the first time which should open the popover, 2. when we click on the popover window to select things) Therefore the e.stopPropagation is needed to exclude those two components from executing the "hide" when clicked on it.